packages/teams-js/src/public/pages.ts

```diff
@@ -130,6 +130,40 @@ export function setCurrentFrameHelper(apiVersionTag: string, frameInfo: FrameInf
   sendMessageToParentWithVersion(apiVersionTag, 'setFrameContext', [frameInfo]);
 }
 
+export function configSetValidityStateHelper(apiVersionTag: string, validityState: boolean): void {
+  ensureInitialized(runtime, FrameContexts.settings, FrameContexts.remove);
+  if (!pages.config.isSupported()) {
+    throw errorNotSupportedOnPlatform;
+  }
+  sendMessageToParentWithVersion(apiVersionTag, 'settings.setValidityState', [validityState]);
+}
+
+export function getConfigHelper(apiVersionTag: string): Promise<pages.InstanceConfig> {
+  return new Promise<pages.InstanceConfig>((resolve) => {
+    ensureInitialized(
+      runtime,
+      FrameContexts.content,
+      FrameContexts.settings,
+      FrameContexts.remove,
+      FrameContexts.sidePanel,
+    );
+    if (!pages.isSupported()) {
+      throw errorNotSupportedOnPlatform;
+    }
+    resolve(sendAndUnwrapWithVersion(apiVersionTag, 'settings.getSettings'));
+  });
+}
+
+export function configSetConfigHelper(apiVersionTag: string, instanceConfig: pages.InstanceConfig): Promise<void> {
```

Comment:
```suggestion
export function setConfigHelper(apiVersionTag: string, instanceConfig: pages.InstanceConfig): Promise<void> {
```

This would make the name match with `getConfigHelper` above

Reply:
hmmm, I think for here, `getConfig` is an API directly under pages capability. But `setConfig` API is the one under pages.config sub capability. That's why they are different. What do you think of the naming pattern of these?

Reply:
since they are helper functions I don't feel strongly about the naming; if you want to leave as-is you can. Go with whatever you think more clearly communicates the correct intent to people reading the code